dd-sdk-android-gradle-plugin/src/main/kotlin/com/datadog/gradle/plugin/kcp/DefaultPluginContextUtils.kt

```diff
@@ -85,5 +110,12 @@ internal class DefaultPluginContextUtils(
         private val composeUiPackageName = FqName("androidx.compose.ui")
         private val modifierClassRelativeName = Name.identifier("Modifier")
         private val modifierThenIdentifier = Name.identifier("then")
+        private val navigationPackageName = FqName("androidx.navigation")
+        private val navHostControllerIdentifier = Name.identifier("NavHostController")
+        private val navHostCallName = FqName("androidx.navigation.compose.NavHost")
+        private val ddTrackEffectPackageName = FqName("com.datadog.android.compose")
+        private val ddTrackEffectIdentifier = Name.identifier("NavigationViewTrackingEffect")
```

Comment:
let's use `datadog` instead of `dd`, this will simplify text-based search in the future

```suggestion
        private val datadogTrackEffectPackageName = FqName("com.datadog.android.compose")
        private val datadogTrackEffectIdentifier = Name.identifier("NavigationViewTrackingEffect")
```